_internal/utils/mutate.ts

```diff
@@ -73,8 +74,7 @@ export async function internalMutate<Data>(
     const keyFilter = _key
     const matchedKeys: Key[] = []
     const it = cache.keys()
-    for (let keyIt = it.next(); !keyIt.done; keyIt = it.next()) {
-      const key = keyIt.value
+    for (const key of it) {
```

Comment:
Needs to revert due to #2181

Reply:
we have already upgraded the build target to es2018 and we also already use for of in https://github.com/vercel/swr/blob/7badbb9d428d2ac48873b3a3f11d54e7cec0c7cf/_internal/utils/cache.ts#L68

I also have checked the build result and it seemed like the for of part was kept.

I guess it won't be a problem here ?

Reply:
should be safe now with es2018. then i think you can just remove the FIXME comment in tsconfig